terraform/modules/lambda_raster_tiler/main.tf

```diff
@@ -26,7 +26,6 @@ resource "aws_lambda_function" "default" {
   tags             = var.tags
   environment {
     variables = {
-      BUCKET    = var.tiles_bucket_name
```

Comment:
I believe we need this info in the lambda. don't we?

Reply:
This was added by me in this commit: https://github.com/wri/gfw-tile-cache/commit/6787070b1019afb79a211fa685dd7410ccf3c076
but it turns out that I needed the bucket var in the fastapi container, not the lambda container. Now I'm trying to clean up.
You're right that the bucket WOULD BE needed in the lambda, but it appears we generate it from the env like so:
    src_tile = f"s3://gfw-data-lake{suffix}/{dataset}/{version}/raster/epsg-3857/zoom_{z}/{pixel_meaning}/geotiff/{str(row).zfill(3)}R_{str(col).zfill(3)}C.tif"

Reply:
Ah, I see. I believe the data lake bucket is created in the core module and we can import it from that state. It would be the cleanest way of doing this.

Reply:
let's do this in a different PR thought. Justing has one open for the tests. It fits better in that PR.